src/_app-theme.scss

```
@@ -42,6 +42,14 @@
     color: mat-color($primary, default-contrast);
   }
 
+  .docs-help-support {
+    color: mat-color($foreground, secondary-text);
```

```suggestion
    color: mat.get-color-from-palette($foreground, secondary-text);
```
This is needed since this PR isn't rebased to pick up the new Sass APIs, but otherwise can be merged cleanly.